src/controllers/notesController.js

```diff
@@ -0,0 +1,103 @@
+import mongooseModel from "../db/schemas/notesSchema.js";
+
+const notesController = {
+  get_all: async (req, res) => {
+    await mongooseModel
+      .find({})
+      .then((result) => {
+        res.json(result);
+      })
+      .catch((err) => {
+        if (err instanceof SyntaxError) {
+          console.error("Error:", err.message);
+          res.send("Error: Wrong type of data, cannot get.");
+        } else {
+          console.error("Error updating note:", err.message);
+          res.send("Error: Something went wrong with getting all the data.");
+        }
+      });
+  },
+
+  post: async (req, res) => {
+    const createTitle = req.body.title;
+    const createContent = req.body.content;
+    const createDate = Date.now();
+
+    await mongooseModel
+      .create({
+        title: createTitle,
+        content: createContent,
+        createAt: createDate,
+        updateAt: "",
```

Comment:
There is no need to pass empty string, field can be undefined at the start of creation